server/domain/feature.js

```diff
@@ -187,23 +187,34 @@ module.exports.getFeature = function(applicationName, featureName, cb) {
             return;
         }
 
-        var metaData = getMetaData(result.node);
-        var isMulti = isMultiFeature(metaData);
-
-        var toggles, toggleSuggestions;
-        if (isMulti){
-            toggles = getMultiFeatureToggles(result.node);
-            toggleSuggestions = getToggleSuggestions(metaData, toggles);
-        } else {
-            toggles = getSimpleFeatureToggle(featureName, result.node);
-        }
+        etcd.client.get('v1/metadata/' + applicationName + '/descriptions', function(descriptionError, descriptionResult){
+            if(descriptionError){
+                console.log(descriptionError);
+            }
+
+            var descriptionsMap = !descriptionError ? getDescriptionsMap(descriptionResult.node) : {};
```

Comment:
Maybe we need to move everything below into a separate method, is sort of getting confusing